features/networking/multus.feature

```diff
@@ -416,4 +416,118 @@ Feature: Multus-CNI related scenarios
+    When I run the :create admin command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/NetworkAttachmentDefinitions/bridge-host-local-novlan.yaml |
+      | n | <%= cb.project_name %>                                                                                                                              |
+    Then the step should succeed
+    #Creating no-vlan pod absorbing above net-attach-def
+    When I run oc create over "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/Pods/generic_multus_pod.yaml" replacing paths:
+      | ["metadata"]["name"] | pod-novlan |
+      | ["metadata"]["annotations"]["k8s.v1.cni.cncf.io/networks"] | bridge3    |
+      | ["spec"]["containers"][0]["name"] | pod-novlan |
+    Then the step should succeed
+    And the pod named "pod-novlan" becomes ready
+    And evaluation of `pod` is stored in the :pod clipboard
+    
+    #Clean-up required to erase bridge interfcaes created on node after this step
+    Given I register clean-up steps:
+    """
+    the bridge interface named "bridge3" is deleted from the "<%= cb.pod.node_name %>" node
+    """
+    When I execute on the pod:
+      | /usr/sbin/ip | -d | link |
+    Then the output should contain "net1"
+    #Entering into corresponding no eot make sure No VLAN ID information shown for secondary interface
+    Given CNI vlan info is obtained on the "<%= cb.pod.node_name %>" node
+    Then the step should succeed
+    And the output should contain 2 times:
+      | 1 PVID untagged |
+
+  # @author anusaxen@redhat.com
+  # @case_id OCP-24489
+  @admin
+  Scenario: Create pod with Multus bridge CNI plugin and vlan tag
+    # Make sure that the multus is enabled
+    Given the multus is enabled on the cluster
+    # Create the net-attach-def via cluster admin
+    Given I have a project
+    And evaluation of `project.name` is stored in the :project_name clipboard
+    When I run the :create admin command with:
+      | f | https://raw.githubusercontent.com/weliang1/Openshift_Networking/master/Features/multus/bridge-host-local-vlan200.yaml |
+      | n | <%= cb.project_name %>                                                                                                |
+    Then the step should succeed
+    #Creating vlan pod absorbing above net-attach-def
+    When I run the :create client command with:
+      | f | https://raw.githubusercontent.com/weliang1/Openshift_Networking/master/Features/multus/pod1-bridge-host-local-vlan200.yaml |
+      | n | <%= cb.project_name %>                                                                                                     |
+    Then the step should succeed
+    And the pod named "pod1-vlan200" becomes ready
+    And evaluation of `pod` is stored in the :pod clipboard
+    
+    #Clean-up required to erase bridge interfcaes created on node after this step
+    Given I register clean-up steps:
```

Comment:
btw instead of wrapping into this step, you can clean-up by using step
```
Given the bridge interface named "mybridge" is deleted from the "<%= cb.pod.node_name %>" node after scenario
```